src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -14,158 +17,146 @@
+                                    Authentication authentication) throws IOException {
+        try {
+            log.info("Добавляем новое объявление: " + properties);
+            return ResponseEntity.ok(adsService.createAd(properties, image, authentication));
+        } catch (IOException e) {
+            log.info("Ошибка при добавлении объявления");
+            return ResponseEntity.status(HttpStatus.UNAUTHORIZED).build();
         }
+    }
 
 
     // получение информации об объявлении
-    @Operation(
-            operationId = "getAdById",
-            summary = "Получение информации об объявлении по id",
-            tags = {"Объявления"},
-            responses = {
-                    @ApiResponse(responseCode = "200", description = "OK"),
-                    @ApiResponse(responseCode = "401", description = "Unauthorized"),
-                    @ApiResponse(responseCode = "404", description = "Not Found")
-            }
-    )
-    @PreAuthorize("hasRole('USER') and @adsService.getAdById(#adId).email == authentication.principal.username")
-
+    @Operation(summary = "Получение информации об объявлении по id")
     @GetMapping("/{adId}")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK"),
+            @ApiResponse(responseCode = "401", description = "Unauthorized"),
+            @ApiResponse(responseCode = "404", description = "Not Found")
+    })
     public ResponseEntity<ExtendedAdDTO> getAdById(@PathVariable int adId) {
         return ResponseEntity.ok().body(adsService.getAdById(adId));
     }
 
     // удаление объявления по id
-    @Operation(
-            operationId = "removeAdById",
-            summary = "Удалить объявление по id",
-            tags = {"Объявления"},
-            responses = {
-                    @ApiResponse(responseCode = "204", description = "No Content"),
-                    @ApiResponse(responseCode = "401", description = "Unauthorized"),
-                    @ApiResponse(responseCode = "403", description = "Forbidden"),
-                    @ApiResponse(responseCode = "404", description = "Not Found")
-            }
-    )
-    @PreAuthorize("hasRole('ADMIN') or (hasRole('USER') and" +
-            "@adsService.getAdById(#adId).email == authentication.principal.username)")
+
+    @Operation(summary = "Удалить объявление по id")
 
     @DeleteMapping("/{adId}")
-    public void removeAdById(@PathVariable int adId) {
-        adsService.deleteAdsById(adId);
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "204", description = "No Content"),
+            @ApiResponse(responseCode = "401", description = "Unauthorized"),
+            @ApiResponse(responseCode = "403", description = "Forbidden"),
+            @ApiResponse(responseCode = "404", description = "Not Found")
+    })
+    @PreAuthorize("hasRole('ADMIN') or " +
+            "@adsService.getAdById(#adId).email == authentication.name")
```

Comment:
```suggestion
            "@adsService.getAdById(#adId).username == authentication.name")
```
потому что поле в сущности User называется username, а не email